src/scss/generic/_typography.scss

```diff
@@ -18,16 +18,10 @@ body {
   }
 }
 
-[hidden] {
+[hidden], [data-hidden] {
   display: none;
 }
 
-[data-hidden] {
```

Comment:
This special ‘hidden’ state only visually hides, to make our form work with password managers that stop working for fields that they recognise as hidden (i.e. set to display: none). LastPass the main password manager we did this for.

If LastPass no longer has a problem with fields set to `display: none` and we find it works with LastPass, let's remove `[data-hidden]` not only from this stylesheet, but also replace `data-hidden` with `hidden` in `ui.js`, where it is being set/toggled, and `index.html`, where it is initially set on a number of cards.

Reply:
Should `inert` be its own PR?

Reply:
From the moment we no longer use the weird `data-hidden` hack, `inert` is no longer needed, so for me keeping it in here would be fine!

Reply:
Added `aria-hidden` to list above. Basically, if LastPass no longer looks at `display: one`, our code will get a whole lot cleaner!